app/src/main/res/layout/activity_join_link.xml

```diff
@@ -1,9 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="70dp"
     tools:context=".ui.activities.JoinLinkActivity">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+
+    <EditText
+        android:id="@+id/insert_room_code"
+        android:layout_width="376dp"
```

Comment:
Using constant values for width/height can be an issue on small / very large screens.
Use match_parent/wrap_content as value with padding / margin for spacing.
I played around with it a little bit, try this:
```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical"
    tools:context=".ui.activities.JoinLinkActivity"
    >
    
    <EditText
        android:id="@+id/insert_room_code"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:background="@drawable/et_style"
        android:padding="10dp"
        android:layout_marginTop="48dp"
        android:layout_marginLeft="16dp"
        android:layout_marginRight="16dp"
        android:hint="@string/insert_room_id"
        android:layout_alignParentTop="true"
        />

    <Button
        android:id="@+id/join_button"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_gravity="center"
        android:backgroundTint="@color/primary"
        android:text="@string/join"
        android:layout_alignParentBottom="true"
        android:layout_margin="16dp"
        />

</RelativeLayout>
```